blocks/graph.py

```diff
@@ -354,3 +384,51 @@ def apply_noise(computation_graph, variables, level, seed=None):
         replace[variable] = (variable +
                              rng.normal(variable.shape, std=level))
     return computation_graph.replace(replace)
+
+
+def apply_dropout(computation_graph, variables, prob=0.5, rng=None, seed=None):
+    """Returns a graph to variables in a computational graph.
+
+    Parameters
+    ----------
+    computation_graph : instance of :class:`ComputationGraph`
+        The computation graph.
+    variables : list of :class:`~tensor.TensorVariable`
+        Variables to be dropped out.
+    prob : float
```

Comment:
I'd recommend naming this more descriptively, such as `drop_prob` or `dropout_prob`, as one can easily slip into thinking about inclusion probabilities, especially with the mistake I've noted below.